src/models/github-server-app.ts

```diff
@@ -50,28 +49,40 @@ export class GitHubServerApp extends EncryptedModel {
 	updatedAt: Date;
 	createdAt: Date;
 
-	getEncryptionSecretKey() {
-		return EncryptionSecretKeyEnum.GITHUB_SERVER_APP;
+	getDecryptedGitHubClientSecret(jiraHost: string): Promise<string> {
+		return this.decrypt(jiraHost, "gitHubClientSecret");
```

Comment:
Just an idea: for the time being, while we haven't run the migration, would it make sense to try-catch and then fallback to the old implementation (without context)?

Reply:
Yup the fallback is in place already. Check inside `this.decrypt()`